infrastructure/main.tf

```diff
@@ -92,4 +97,16 @@ resource "azurerm_key_vault_secret" "smoke-test-s2s-url" {
   value     = "${var.s2s_url}"
   vault_uri = "${module.key-vault.key_vault_uri}"
 }
+
+resource "azurerm_key_vault_secret" "smoke-test-s2s-name" {
+  name      = "smoke-test-s2s-name"
+  value     = "send_letter_tests"
```

Comment:
Is this service meant to be used in both functional and smoke tests?
If so, it would be worth renaming the secret.